cypress/integration/common/selectors-and-assertions.spec.ts

```diff
@@ -50,10 +50,21 @@ export const iCanSeeASectionHeadingThatContains = (text: string): void => {
   cy.get("h2").contains(text);
 };
 
+export const iCanSeeNLinksContaining = (n: number, text: string): void => {
+  cy.get(`a[href]:contains(${text})`).should("have.length", n);
+};
+
+export const iCanSeeAButtonContaining = (text: string | RegExp): void => {
+  cy.get(`[role=button]:contains(${text})`);
+};
+
 export const givenIHaveClickedTheButtonContaining = (text: string): void => {
-  cy.get("button").contains(text).click();
+  cy.get(`[role=button]:contains(${text})`).click();
```

Comment:
To allow for Cypress to get `a href`s that use the ARIA role of `button`, in particular those "Continue" links that are for all intents and purposes buttons do not appear inside a `<form>`.